bundles/org.openhab.binding.govee/src/main/java/org/openhab/binding/govee/internal/CommunicationManager.java

```diff
@@ -195,13 +217,13 @@ void stopReceiving() {
         public void run() {
             while (!stopped) {
                 try {
-                    socket = new MulticastSocket(RESPONSE_PORT);
+                    socket = new DatagramSocket(RESPONSE_PORT);
```

Comment:
Assign this to a non-nullable local variable first and to member variable later and use the former below? Gets rid of the ugly requireNonNull calls.

Reply:
Ok.